index.html

```diff
@@ -220,7 +220,53 @@ <h3>Skills</h3>
     </main>
 
     <footer>
+        <h3 class="footer_title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        <form action="https://formspree.io/f/xgebnwyd" method="POST" class="flex_column">
+            <ul class="flex_column">
+                <li class="form_list">
+                    <input 
+                    type="text" 
+                    id="name" 
+                    name="user_name" 
+                    placeholder="Full name" 
+                    minlength="2" 
+                    maxlength="30" 
+                    required>
+                </li>
+                <li class="form_list">
+                    <input 
+                    type="email" 
+                    name="user_email" 
+                    id="email" 
+                    placeholder="Email address" 
+                    required>
+                </li>
+                <li>
+                    <textarea 
+                    name="user_message" 
+                    id="message" 
+                    placeholder="write something..." 
+                    minlength="2" 
+                    maxlength="500" 
+                    required></textarea>
+                </li>
+                <li class="form_button flex">
+                    <button type="submit" class="button_enabled">Get in touch</button>
```

Comment:
[OPTIONAL]
- [ ] Kindly remove the contact form from the `footer` tag for semantics reasons.

- A `<footer>` element typically contains:
  - authorship information
  - copyright information
  - contact information
  - sitemap
  - back to top links
  - related documents

Reply:
Man, really